compiler/ir/backend.js/src/org/jetbrains/kotlin/ir/backend/js/compiler.kt

```diff
@@ -142,7 +144,7 @@ fun compile(
 
     val deserializedModuleFragments = sortedImmediateDependencies.map {
         val moduleFile = File(it.klibPath, moduleHeaderFileName)
-        deserializer.deserializeIrModuleHeader(depsDescriptors.getModuleDescriptor(it), moduleFile.readBytes(), File(it.klibPath), DeserializationStrategy.ONLY_REFERENCED)
+        deserializer.DeserializeIrModuleHeader(depsDescriptors.getModuleDescriptor(it))
```

Comment:
deserializer.~D~deserializeIrModuleHeader

Reply:
fixed, it was a rebase artefact